src/components/common/Tabs/Tabs.tsx

```diff
@@ -5,28 +5,31 @@ import styled from 'styled-components'
 import TabItem from 'components/common/Tabs/TabItem'
 import TabContent from 'components/common/Tabs/TabContent'
 
-export interface TabItemType {
-  readonly id: number
-  readonly title: string
+type TabId = number
+
+export interface TabItemInterface {
```

Comment:
Why a type need a sufix `Type` or `Interface` at the end? is `TabItem` not good?

Reply:
`TabItem` as a component already exists, was getitng name clash